src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingMethodSymbol.cs

```diff
@@ -219,6 +222,43 @@ public override ImmutableArray<CSharpAttributeData> GetReturnTypeAttributes()
             return this.RetargetingTranslator.GetRetargetedAttributes(_underlyingMethod.GetReturnTypeAttributes(), ref _lazyReturnTypeCustomAttributes);
         }
 
+#nullable enable
+        internal override UnmanagedCallersOnlyAttributeData? UnmanagedCallersOnlyAttributeData
+        {
+            get
+            {
+                if (_lazyUnmanagedAttributeData == UnmanagedCallersOnlyAttributeData.Uninitialized)
+                {
+                    var underlyingData = _underlyingMethod.UnmanagedCallersOnlyAttributeData;
+                    if (underlyingData == UnmanagedCallersOnlyAttributeData.Uninitialized)
+                    {
+                        // Underlying hasn't been found yet either, just return it. We'll check again the next
+                        // time this is called
+                        return underlyingData;
+                    }
+
+                    if (underlyingData == null || (underlyingData.CallingConventionTypes?.IsEmpty != false))
```

Comment:
>if [](start = 20, length = 2)

Consider writing the `if` so there's only one call to `Interlocked.CompareExchange()` in this method.